docs/how-payjoin-saves-exchanges.md

```diff
@@ -0,0 +1,117 @@
+
+Transactions compete to get included in blocks according to network fees they pay since each block is limited to a fixed weight. At a high level, each transaction weight pays some base costs (𝑏), per-input costs (𝑖) and per-output costs (𝑜). In reality not all inputs and outputs have equal cost but the principle can be explained assuming they are, and be backed up by examples of real transaction. For example, a simple exchange could have 5 btc in their treasury and sell 1 bitcoin each to Alice, bob, and Carol.
+
+```
+Exchange 5 btc  ->  1 BTC to Alice
+                   ~4 BTC minus fees to Exchange
+```
+
+```
+Exchange 4 btc  ->  1 btc to Bob
+                   ~3 BTC minus fees to Exchange
+```
+
+```
+Exchange 3 btc  ->  1 btc to Carol
+                   ~2 BTC minus fees to Exchange
+```
+
+Each transaction would cost the exchange 𝑏 + 𝑖 + 2𝑜, and they would pass the fees onto their customers in order to make a profit. The sum of these costs would be 3𝑏 + 3𝑖 + 6𝑜, which would come out of the final ~2 BTC change the exchange makes at last.
+
+## Payment Batching
+
+Batching helps the exchange save time and money in two ways. First, the overall cost to post such a transaction is cheaper than the sum of three individual transactions to produce the same result. Second, a single unspent output can fund multiple withdrawals.
+
+your typical exchange withdrawal looks like this:
+
+```
+Exchange 5 btc  ->   1 btc to Alice
+                     1 btc to Bob
+                     1 btc to Carol
+                    ~2 BTC minus fees to Exchange
+```
+
+All else equal, an exchange making this batch instead of three separate transactions will only pay 𝑏 + 𝑖 + 4𝑜 in fees, saving 2𝑏 + 2𝑖 + 2𝑜 compared to 3𝑏 + 3𝑖 + 6𝑜, at least 33% cheaper than making those three transactions separately. Second, the exchange does not have to wait for each withdrawal to settle before paying out the next one since they can service them all with a single UTXO. Sure, the exchange could keep multiple UTXOs ready for spending, but that is always going to be overhead of even more transactions in preparation.
+
+## Payjoin Payment Batching
+
+What if Dave the depositor can payjoin? He gets a benefit of breaking some privacy heuristics, and he can save the exchange some fees that might be able to be pased on to him. Payjoin lets the exchange fund withdrawals with Dave's deposit in the same transaction.
+
+```
+Dave     3 btc  ->  1 btc to Alice
+Exchange 2 btc      1 btc to Bob
+                    1 btc to Carol
+                   ~2 BTC minus fees to Exchange
+```
+
+𝑏 + 2𝑖 + 4𝑜
```

Comment:
This just assumes that the exchange chooses to batch 1 UTXO. Theoretically there is no limit to how many UTXOs could be batched here? In which case the formula could be more general to account for the number of Exchange/receiver-added UTXOs?